install/update_945_946.php

```diff
@@ -30,50 +30,29 @@
  * ---------------------------------------------------------------------
  */
 
-if (!defined('GLPI_ROOT')) {
-   die("Sorry. You can't access this file directly");
-}
-
 /**
- * Backup class
+ * Update from 9.4.5 to 9.4.6
  *
- * @since 0.85
+ * @return bool for success (will die for most error)
 **/
+function update945to946() {
+   global $DB, $migration;
 
+   $updateresult     = true;
 
-class Backup extends CommonGLPI {
```

Comment:
Should this Backup class deprecated in 9.5.0 before being dropped completely ?
At least, an entry in changelog should be added.

Reply:
No, this class was not really used (all the logic was in the front page).